projects/plugins/boost/app/assets/src/js/utils/add-get-parameter.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Helper function to add a get parameter to the end of the supplied URL.
+ *
+ * @param {string} url   - URL to add the parameter to.
+ * @param {string} key   - Parameter key.
+ * @param {string} value - Parameter value.
+ * @return {string} - URL with the parameter added.
+ */
+export function addGetParameter( url: string, key: string, value: string ): string {
```

Comment:
This is probably fine in this use case. However, since it will stay in utils and potentially be reused, I think would be good to handle corner cases.

This may fail if the passed URL has a hash in the URL. I think a good approach would be to pass the URL through `URL` class in js and modifying the query string there.

Reply:
I didn't, because I was a little concerned that `URL` can be finicky about parsing invalid URLs and will cause the tool to fail.

But that should be a rare case, so I've gone with the "correct" solution, as you suggested in a03c98b662